### PR TITLE
Documentation/edd20 hash update how to

### PR DESCRIPTION
**Pull request by cjjohansen**
**Description of your changes**
Added explanation on how to update dd20 format hash values

Category:

- [ ] New Feature
- [ ] Bugfix
- [x] Documentation update

Checklist:

- [x] I have bumped all relevant version numbers
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have documented my changes if this is needed
- [x] I have described my changes in the "description of your changes" field
- [ ] I have tested this change on a running cluster 
- [x] I have created/corrected automated tests related to the changes, if possible/necessary
- [x] I have checked that new and existing unit tests pass locally with my changes
- [x] I have squashed commits if necessary

### CODE REVIEW

File: chart/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: conductor-data-provider
 description: A helm chart for energinet-singularity/conductor-data-provider
-version: 1.2.2
+version: 1.2.3
```
Comment (CodeWitten) [inline chart/Chart.yaml:4]: bump to 1.2.4 as version in main is not 1.2.3

File: chart/values.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ replicaCount: 1
 image:
   repository: ghcr.io/energinet-singularity/conductor-data-provider/energinet-singularity/conductor-data-provider
   pullPolicy: IfNotPresent
-  tag: "1.2.2"
+  tag: "1.2.3"
```
Comment (CodeWitten) [inline chart/values.yaml:9]: bump to 1.2.4 as version in main is not 1.2.3

File: README.md [hunk 3]
```diff
@@ -120,3 +120,24 @@ Please submit an issue or ask the authors.
...
+In case the format of the headers change, we will trigger an exception.
+
+To update the expected hash value, which will be needed once correction has been made to handle a new dd20 format.
+One has to run the hash calculation and store the value as the new expected hash value.
+
+This can be done by following these steps.
+
+* 1. Save the new dd20 file in a sub folder under the tests folder. For example
+ > \tests\real-data\new-dd20.XLSM
+* 2. update your local .env file under app folder to contain (You cna create one if it doesnn't exist)
```
[the remark this answers is not in the document]
Comment (cjjohansen) [reply README.md:136]: Will fix

File: README.md [hunk 4]
```diff
@@ -120,3 +120,24 @@ Please submit an issue or ask the authors.
...
+> MOCK_DD20_FILEPATH="real-data/new-dd20.XLSM
+* 3. run the test test_can_validate_dd20_line_data_summer_format. It should fail. And you should be able to see the old and the new hash value in the assertion message
+* 4. search and replace the old value with the new value in the code base.
```
[the remark this answers is not in the document]
Comment (cjjohansen) [reply README.md:139]: Well the idea is that the DD20Settings class reads from env variables and from .env file. So specifying in .env file is sufficient.
Does it answer your question?

File: chart/values.yaml (hunk 2 above)
Comment (cjjohansen) [reply chart/values.yaml]: fixed

File: chart/Chart.yaml (hunk 1 above)
Comment (cjjohansen) [reply chart/Chart.yaml]: fixed

**Review by CodeWitten — APPROVED** [review]

**Review by CodeWitten — APPROVED** [review]

**Review by CodeWitten — APPROVED** [review]

**Pull request merged**